controllers/clusters/postgresql_controller.go

```diff
@@ -569,6 +569,129 @@ func (r *PostgreSQLReconciler) createUser(
 	return nil
 }
 
+func (r *PostgreSQLReconciler) handleUsersDelete(
+	ctx context.Context,
+	l logr.Logger,
+	c *v1beta1.PostgreSQL,
```

Comment:
Why do you refer to PostgreSQL as `c`? Let's use `pg` at least, wdyt?

Reply:
It is like cluster, but ok, renamed